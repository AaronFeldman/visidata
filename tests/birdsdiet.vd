sheet	col	row	longname	input	keystrokes	comment
			open-file	sample_data/birdsdiet.tsv	o	open input in VisiData
birdsdiet	Family		key-col		!	pin current column on the left as a key column
birdsdiet	MaxAbund		hide-col		-	hide current column
birdsdiet	AvgAbund		hide-col		-	hide current column
birdsdiet	Mass		type-float		%	set type of current column to float
birdsdiet	Passerine		select-col-regex	1	|	select rows matching regex in current column
birdsdiet			dup-selected		"	open duplicate sheet with only selected rows
birdsdiet_selectedref	Mass		aggregate-col	mean	+	add aggregator to the current column
birdsdiet_selectedref	Diet		freq-col		F	open Frequency Table grouped on current column
birdsdiet_selectedref_Diet_freq	mean_Mass		sort-desc		]	sort descending by current column
birdsdiet_selectedref_Diet_freq	count		hide-col		-	hide current column
