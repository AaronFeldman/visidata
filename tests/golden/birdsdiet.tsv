Diet	mean_Mass
InsectVert	55.86
PlantInsect	36.78
Plant	21.81
Insect	11.28
